doc/classes/CPUParticles2D.xml

```diff
@@ -46,6 +46,13 @@
 				Returns the enabled state of the given flag (see [enum ParticleFlags] for options).
 			</description>
 		</method>
+		<method name="request_particles_process">
+			<return type="void" />
+			<param index="0" name="process_time" type="float" />
+			<description>
+				Requests the particles to process for extra process time during a single frame. Used for seeking in particles playback.
```

Comment:
The part about particles playback can be confusing, as no such thing exists currently. You can instead mention how it can be used, something like "In combination with [method restart], it can be used for particle seeking when [member speed_scale] is [code]0[/code].".